### DISCUSSION TITLE
Default Style

### DISCUSSION THREAD

**Discussion by hxhgxy** [ControlzEx/ControlzEx#185] [Q&A] [answered]
![ControlzEx](https://github.com/ControlzEx/ControlzEx/assets/37101628/c0347d2c-52e8-491a-8975-736ceff1b3fc)
I tried to delete the window style key and make a default style. the window is black.

There should be a way to do that, right?

**Comment by batzen** [comment, marked as answer]
That does not work in WPF and is not an issue of ControlzEx.

If you want to define the default style for a window class that has to happen in the source assembly of the window class itself, as far as I remember.

**Comment by hxhgxy** [comment]
I tried to do it in Window.xaml and it doesn't work. In previous version, I can do it in window.xaml (in .net framework 4.8 projects).
Anyway this is not a big problem.

**Reply by batzen** [reply]
What do you mean with "previous version"?
